woocommerce/woocommerce
# put products list suggestions on hiatus (month) after multiple dismissals

### All Submissions:




### Changes proposed in this Pull Request:
- Track the number of dismissals of product-list suggestions in a cookie.
- If this exceeds 5, don't show any product list suggestions.
- Suggestions will resume after a month, as the cookie expires after 31 days.

### How to test the changes in this Pull Request:
1. Get this branch running on your Woo dev setup.
2. Visit products admin page, with some products set up (not empty state).
3. See a suggestion in the products list.
2. Dismiss the suggestion - `woocommerce_dismissed_suggestions__products-list-inline` cookie should increase by 1.
1. Delete the cookie `woocommerce_snooze_suggestions__products-list-inline`, or wait 24 hours :)
1. Refresh the page, and keep dismissing suggestions until you have dismissed 5.
1. You should not see any suggestions in product list when `woocommerce_dismissed_suggestions__products-list-inline` > 5.

If you need to reset your dismissed suggestions, you can do so by clearing user meta key `wc_marketplace_suggestions_dismissed_suggestions`, e.g. `wp user meta delete YOUR_USERNAME wc_marketplace_suggestions_dismissed_suggestions`.
